handlers/validator.go

```diff
@@ -1177,7 +1177,7 @@ func ValidatorAttestations(w http.ResponseWriter, r *http.Request) {
 		totalCount = 0
 	}
 	lastAttestationEpoch := epoch
-	if ae.ExitEpoch != 9223372036854775807 {
+	if ae.ExitEpoch != 9223372036854775807 && ae.ExitEpoch < epoch {
 		lastAttestationEpoch = ae.ExitEpoch
```

Comment:
As discussed this would include the exit epoch in the attestations table for which no data exists.
As a result the first page for all validators that have already existed show only 9 entries.
Further pages are fine.

Maybe this change is enough
```suggestion
		lastAttestationEpoch = ae.ExitEpoch - 1
```